packages/govuk-frontend/src/govuk/components/checkboxes/_index.scss

```diff
@@ -29,39 +24,42 @@
   }
 
   .govuk-checkboxes__input {
-    $input-offset: ($govuk-touch-target-size - $govuk-checkboxes-size) / 2;
-
-    position: absolute;
-
-    z-index: 1;
-    top: $input-offset * -1;
-    left: $input-offset * -1;
-
     width: $govuk-touch-target-size;
     height: $govuk-touch-target-size;
     margin: 0;
-
     opacity: 0;
-
     cursor: pointer;
   }
 
   .govuk-checkboxes__label {
-    display: inline-block;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+
+    // Ensure that the width of the label is never more than the width of the
+    // container minus the input width minus the padding on either side of
+    // the label. This prevents the label from going onto the next line due to
+    // __item using flex-wrap because we want hints on a separate line.
+    max-width: calc(100% - (($govuk-checkboxes-label-padding-left-right * 2) + $govuk-touch-target-size));
     margin-bottom: 0;
-    padding: 8px $govuk-checkboxes-label-padding-left-right govuk-spacing(1);
+    padding: (govuk-spacing(1) + $govuk-border-width-form-element) govuk-spacing(3);
     cursor: pointer;
     // remove 300ms pause on mobile
     touch-action: manipulation;
   }
 
+  // to account for when there are multiple children in a label
+  .govuk-checkboxes__label * {
```

Comment:
Mainly mentioned [labels with `flex-wrap` above](https://github.com/alphagov/govuk-frontend/pull/4093/files#r1302864343) since `label.html` with child `<strong>` would show:

<img width="585" alt="Checkbox label children issue" src="https://github.com/alphagov/govuk-frontend/assets/415517/e0e1163d-f292-4039-9b93-e9ad2d6de57f">

Reply:
Bugger. If you remove the `width: 100%` you'll see this most prominently in [this example](https://govuk-frontend-pr-4093.herokuapp.com/components/radios/with-legend-as-page-heading/preview). We probably need to retain the inline/block level styling of the label's children, but that makes it difficult to automatically centrally align. I'll have a muck around.

Reply:
Yeah, was thinking for safety it might be back to tweaking `padding-top` 😬

Reply:
I believe I've solved this by removing `flex` from the label and using `align-self: center`